compiler/bin-js_of_ocaml/compile.ml

```diff
@@ -172,6 +173,12 @@ let run
   | `Name _, _ -> ());
   List.iter params ~f:(fun (s, v) -> Config.Param.set s v);
   List.iter static_env ~f:(fun (s, v) -> Eval.set_static_env s v);
+  (* For backward compatibility, consider that [--enable effects] alone means
+     [--effects cps] *)
+  Config.set_effects_backend
```

Comment:
Maybe move the set call inside the match branch

Reply:
Ignore the request. I misread the whole logic

Reply:
I think that this whole backward compatibility logic should be moved to the cmd_arg module.

Reply:
It’s non-trivial because `Config.Flag.effects ()` is initialized by some separate logic, not in `Cmd_arg`. But I can try.

Reply:
Indeed, it is initialized by Jsoo_cmdline.Arg.eval.